fonts.css

```diff
@@ -0,0 +1,34 @@
+@font-face {
```

Comment:
I created this separate fonts.css file for organization purposes and cleanliness 🙂